packages/patterns/test/copySet.test.js

```diff
@@ -119,6 +138,23 @@ test('operations', t => {
   t.deepEqual(yIz, makeTagged('copySet', ['b']));
 });
 
+test('setIsSuperset', async t => {
+  await fc.assert(
+    fc.property(
+      fc.uniqueArray(arbKey, { comparator: keyEQ }),
+      fc.infiniteStream(fc.boolean()),
+      (arr, keepSeq) => {
+        const sub = arr.filter(() => keepSeq.next().value);
+        setIsSuperset(makeCopySet(arr), makeCopySet(sub)) ||
+          Fail`${q(sub)} of ${q(arr)}`;
+      },
+    ),
+  );
```

Comment:
I’m not a qualified reviewer for property tests yet.

Reply:
I read this as: For an arbitrary array of distinct keys `arr` and subsequence `sub` of `arr`, `setIsSuperset(makeCopySet(arr), makeCopySet(sub))` holds, where a subsequence is formed by filtering using elements of an arbitrary infinite stream of booleans.

Reply:
I thought it would be illustrative to show the output when it fails, before the fix is applied. But even when I check out 14fca7b49 that has the test and not the fix, this test seems to pass. I suppose the failing case is pretty obscure and the generator doesn't get that far?

When I crank `numRuns` up to 4000, I get a failure that might be showing another problem? `Got error: Error: ["[undefined]"] of ["[Symbol()]",null,"[undefined]"]`

Reply:
oic... it's finding cases of the `undefined` problem.